apps/src/componentLibrary/toggle/Toggle.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import classnames from 'classnames';
+import Typography from '@cdo/apps/componentLibrary/typography';
+import moduleStyles from './toggle.module.scss';
+
+interface ToggleProps {
+  checked: boolean;
+  onChange: () => void;
+  name: string;
+  value?: string;
+  label?: string;
+  disabled?: boolean;
+  position?: 'left' | 'right';
+  size?: 'xs' | 's' | 'm' | 'l';
+}
```

Comment:
Whoa, Typescript! I haven't used TS in a while, so I'm not confident enough to give a full review. But I'll look at what I can. Really excited about how using TS will allow us to build out the design library more clearly 🙌

Reply:
Yeah, we're using TS in Student Learning team in some places and I hope we'll be using it more across our codebase in the future 😄